app/lambda/get_credentials/lambda_function.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+import sys
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+
+
+class GetCredentialsLambda(LambdaBase):
+    def __init__(self): # implementation-specific args and/or kwargs
+        pass
+
+    def handle(self, event, context):
+        print("event: {}".format(event))
+        sts = boto3.client('sts')
```

Comment:
Should this also use the AWS Helper to get the boto client?

Reply:
Done.

Reply:
Added comment on another change  on the new PR
https://github.com/aws-samples/medical-transcription-analysis/pull/21/files